web/ui/module/codemirror-promql/src/grammar/test/expression.txt

```diff
@@ -846,4 +846,10 @@ MetricName(MetricIdentifier(Identifier))
 1 + foo atan2 bar
 
 ==>
-PromQL(Expr(BinaryExpr(Expr(NumberLiteral),Add,BinModifiers,Expr(BinaryExpr(Expr(VectorSelector(MetricIdentifier(Identifier))),Atan2,BinModifiers,Expr(VectorSelector(MetricIdentifier(Identifier))))))))
\ No newline at end of file
+PromQL(Expr(BinaryExpr(Expr(NumberLiteral),Add,BinModifiers,Expr(BinaryExpr(Expr(VectorSelector(MetricIdentifier(Identifier))),Atan2,BinModifiers,Expr(VectorSelector(MetricIdentifier(Identifier))))))))
+# Evaluation aligned variant of a subquery
```

Comment:
Could you add a test with the current syntax for the subquery please (i.e with a single `:`) ? It's just to be sure that one syntax is not ignored by lezer

Reply:
Sure!